docs/source/getting_started.ipynb

```diff
@@ -64,6 +64,32 @@
     "        return index_to_char"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "nbsphinx": "hidden"
+   },
+   "outputs": [],
+   "source": [
+    "file_content = \"\"\"\\\n",
+    "from typing import Dict\n",
+    "\n",
+    "\n",
+    "class C(object):\n",
+    "    attr = \"ab\" \"cd\" \"ef\" \"gh\"\n",
+    "\n",
+    "    def method(self) -> Dict[int, str]:\n",
+    "        filtered_char = []\n",
+    "        for char in self.attr:\n",
+    "            if char is not \"a\":\n",
+    "                filtered_char.append(char)\n",
+    "\n",
+    "        index_to_char = dict([(idx, char) for idx, char in enumerate(filtered_char)])\n",
+    "        return index_to_char\n",
+    "\"\"\""
+   ]
+  },
```

Comment:
Copy the example code is not ideal. It'll make the future updated less easy.

I've find a workaround in https://github.com/Instagram/Fixit/pull/133

Reply:
I'll revert it. Thanks